xc/xc7/toolchain_wrappers/synth

```diff
@@ -6,10 +6,15 @@ MYPATH=`dirname ${MYPATH}`
 
 export TECHMAP_PATH=`realpath ${MYPATH}/../share/techmaps/xc7_vpr/techmap`
 
-SYNTH_TCL_PATH=`realpath ${MYPATH}/../share/prjxray/synth.tcl`
-CONV_TCL_PATH=`realpath ${MYPATH}/../share/prjxray/conv.tcl`
+SYNTH_TCL_PATH=`realpath ${MYPATH}/../share/xc7/synth.tcl`
+CONV_TCL_PATH=`realpath ${MYPATH}/../share/xc7/conv.tcl`
```

Comment:
The name `/usr/share/symbiflow/xc7/scripts/XXXX.tcl` might be better.

Reply:
Done